browser/lib/transport/xhrrequest.js

```diff
@@ -8,6 +8,14 @@ import Defaults from '../../../common/lib/util/defaults';
 import BufferUtils from 'platform-bufferutils';
 import DomEvent from '../util/domevent';
 
+function getAblyError(responseBody, headers) {
+	if (headers['x-ably-errorcode'] | headers['X-Ably-ErrorCode']) {
```

Comment:
I think HTTP headers are case insensitive. Can't you just do a case insensitive compare to catch all variants?

Reply:
^ what quintin said